README.md

```diff
@@ -86,6 +86,7 @@ db_port: 6000
 db_user: {unid}
 ```
 
+Additionally, you can set the environment variable `DB_RESET=false` or add `db_reset: false` to `valis.yaml`. This will prevent the DB connection to be closed after a query completes and should speed up new queries. This setting should not be used in production.
```

Comment:
the environment variable in this case would be `VALIS_DB_RESET`.  All of the attributes in `Settings` are prefixed with `VALIS_` for use as envvars.  
```suggestion
Additionally, you can set the environment variable `VALIS_DB_RESET=false` or add `db_reset: false` to `valis.yaml`. This will prevent the DB connection to be closed after a query completes and should speed up new queries. This setting should not be used in production.
```

Reply:
Fixed.